### ISSUE TITLE
RPi 4B support ?

### ISSUE THREAD

**Issue by luczjanoo** [g0orx/pihpsdr#113]
Hi.

Does piHPSDR working on RPi 4B ?

**Comment by g0orx** [comment]
Yes, that is the preferred RPi because of the improved performance.

**Comment by luczjanoo** [comment]
Thanks for info. Just tried precompiled binaries 2.0.0rc2 but without success. I've got error with wiringpi and gpio.

Also tried to compile latest sources but I can't find a list of dev packages to be installed before. I've tried to install missed one on every error but many work to do :)

Maybe could you put release ready to work on rpi4?

**Comment by gerryk** [comment]
Thanks @g0orx 
Will the 2GB unit suffice?
